test/imports/LunaNext/Compositor/Singletons/WindowModelSingleton.qml

```diff
@@ -39,7 +39,7 @@ Item {
     Connections {
         target: _compositor
 
-        onWindowAddedInListModel: {
+        function onWindowAddedInListModel() {
```

Comment:
it's `signal windowAddedInListModel(Item window);`
so you need  `function onWindowAddedInListModel(window)`